packages/netlify-cms-backend-github/src/API.js

```diff
@@ -710,16 +735,14 @@ export default class API {
    * Get a pull request by PR number.
    */
   getPullRequest(prNumber) {
```

Comment:
This method was not used anywhere, and in any case I think it should use `originRepoURL`. 
I fixed the API call to use `originRepoURL` and used it in `updateUnpublishedEntryStatus` and `getUpdatedForkWorkflowMetadata`

Reply:
Fixed in #2523: https://github.com/netlify/netlify-cms/commit/9a63d11a9be15d0cd8c5a71141774b2175d9b284#diff-b4ba42c688a23ed9cec5139daec2c157R735-R738